Cargo.toml

```diff
@@ -10,7 +10,14 @@ license = "Apache-2.0/MIT"
 categories = ["concurrency", "data-structures"]
 
 [features]
-nightly = []
+default = ["use_std"]
+nightly = ["crossbeam-epoch/nightly"]
+use_std = ["crossbeam-epoch/use_std"]
+
+[dependencies]
+crossbeam-epoch = "0.2"
+crossbeam-utils = "0.2"
+crossbeam-deque = "0.1"
```

Comment:
Do we not want to expose `channel` for now?

Reply:
Eventually, `crossbeam-channel` should replace the two queue implementations in `sync`.  I don't know what's the best way to do it.  I omitted this migration in this PR because it's @stjepang's call to make.